department-of-veterans-affairs/caseflow
# Enable pagination filtering by assignee

### Description
Enable pagination filtering by assignee

|Before|After 1|After 2|
|---|---|---|
|<img width="918" alt="Screen Shot 2019-09-13 at 11 31 32 AM" src="https://user-images.githubusercontent.com/45575454/64875052-3f3d8180-d61a-11e9-9201-d1ed56c5b82b.png">|<img width="900" alt="Screen Shot 2019-09-13 at 11 32 06 AM" src="https://user-images.githubusercontent.com/45575454/64875060-42d10880-d61a-11e9-9b9d-651ad2c10068.png">|<img width="871" alt="Screen Shot 2019-09-13 at 11 32 36 AM" src="https://user-images.githubusercontent.com/45575454/64875063-449acc00-d61a-11e9-8689-755cf5a43fe4.png">|

### Acceptance Criteria
- [ ] Can filter by assignee when pagination is turned on

### Testing Plan
1. Turn on pagination
2. Ensure filtering by assignee works
